qucs/README

```diff
@@ -36,6 +36,8 @@ Requirements
 
 Qucs needs Qt(R) by Digia(R) <http://www.digia.com>.
```

Comment:
not related but Qt development is now done by the Qt Group Plc, https://www.qt.io :grin: